test/parallel/test-buffer-readint16be.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readInt16BE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readInt16BE(1);
+
+assert.strictEqual(result, 21672);
```

Comment:
nit: I think using hex constants inside of `Buffer.from` and in `strictEqual` would make the test a bit more “obviously correct”

i.e. `Buffer.from([0x2a, 0x54, 0xa8, 0x7f])` and `assert.strictEqual(result, 0x54a8);`

(this applies to the other readInt*/readUInt* too)